src/components/Dropdown.js

```diff
@@ -94,8 +93,8 @@ export default class Dropdown extends React.Component {
     if (this.refs.list) this.refs.list.handleKeys(event)
   }
 
-  chooseChild = choice => {
-    const link = findChildLink(findDOMNode(this.refs[choice.ref]))
+  chooseChild = (element, node) => {
+    const link = findChildLink(node)
```

Comment:
we need it. findChildLink takes the original DOM node, (which will be an
<li>) and gets the embedded <a> which it then returns to be clicked. If we
send a click to the top level <li>, nothing happens

On Tue, Aug 30, 2016 at 3:53 PM, Lawrence Wang <notifications@github.com>
wrote:

> In src/components/Dropdown.js
> <https://github.com/Hylozoic/hylo-redux/pull/111#discussion_r76886898>:
>
> > @@ -94,8 +93,8 @@ export default class Dropdown extends React.Component {
> >      if (this.refs.list) this.refs.list.handleKeys(event)
> >    }
> >
> > -  chooseChild = choice => {
> > -    const link = findChildLink(findDOMNode(this.refs[choice.ref]))
> > +  chooseChild = (element, node) => {
> > +    const link = findChildLink(node)
>
> findChildLink should be unnecessary now, no?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Hylozoic/hylo-redux/pull/111/files/603a4a5abc4ec8a8d1eafab63399d1e6bff4d5d5..9f3229dc56955e8e54525c34244be2ead76fef89#r76886898>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA2Y9FztMda4GcFcaLrff0qHdL2t4DTuks5qlKZcgaJpZM4Jv_H6>
> .
>



-- 
Robbie Carlton

719.358.3482
<robbie@authenticmanprogram.com>